lib/hammer_cli_katello/content_view_purge.rb

```diff
@@ -59,6 +52,7 @@ def execute
       end
 
       # Check if there is something to do
+      old_unused_versions = purge_list
```

Comment:
I'm not sure I understand why you rewrote the method instead of altering the existing method. The old method cached the result in an instance variable for reuse in subsequent calls, so a variable wasn't necessary here. :man_shrugging:

Reply:
I guess I preferred to make one resource call rather than 2. The existing method required a call to both `content_view` and `content_view_version` resources.

Reply:
I guess I preferred to make one resource call rather than 2. The existing method required a call to both content_view and content_view_version resources, which was unnecessary.

Reply:
Yeah, you'd still need to alter the API call, but there is some logic that you want to keep (like sorting by version, which you had to re-implement in your method). So I'm wondering why you chose to rewrite it instead of altering it. When I submit a code change, I generally try to modify the existing code as little as possible (make the diff as small and easy to understand as possible, keep things that are already working correctly, etc.). This helps people in the future when they're looking at this commit to figure out what changed. I'd prefer you modify the original method rather than replace it for that reason.

Reply:
Honestly, I chose to rewrite it because it seemed messy. I didn't see any reason to keep an instance variable, as we'd want to refresh the information any time this method is called. I've made the requested changes. Thanks for explaining.

Reply:
Yup, that is why I thanked you for explaining and made the changes 👍